src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -337,48 +410,55 @@ export default class PhotoField extends React.Component {
     this.setState({ dragAndDropSupported: dragAndDropSupported && !iOS });
   }
 
-  updateBoundaryWarningAndButtonStates = () => {
-    const moveBoundariesMet = getMoveBoundariesMet(this.refs.cropper.getCanvasData(), this.refs.cropper.getCropBoxData());
-
+  updateBoundaryWarningAndButtonStates = (moveBoundariesMet, zoomWarn = false) => {
     this.setState({
-      warningMessage: makeCropBoundaryWarningMessage(moveBoundariesMet),
+      warningMessage: makeCropBoundaryWarningMessage({ zoomWarn, ...moveBoundariesMet }),
       ...makeMoveButtonsEnabledStates(moveBoundariesMet)
     });
   }
 
   handleCropend = () => { // casing matches Cropper argument
-    this.updateBoundaryWarningAndButtonStates();
+    this.maybeMoveCanvasWithinBounds();
+  }
+
+  handleCropstart = (e) => {
```

Comment:
@jbalboni Alrighty I got the black border to display again. The component "disables" moving the cropbox edges by `display: none`ing the border with css rather than by limiting the actions in js. I re-enabled the draggable cropbox but block the drag action from propagating in `handleCropstart`.